Readme.md

```diff
@@ -0,0 +1,9 @@
+## Two Truths and a Lie
+
+I wear a ring
+
+I can't log into eduroam wifi
+
+I'm going to California after this course
```

Comment:
https://github.com/huangyz97/TwoTruths/pull/1

On Fri, Jun 17, 2016 at 10:31 AM, idelgiudice001 <notifications@github.com>
wrote:

> In Readme.md
> <https://github.com/huangyz97/TwoTruths/pull/1#discussion_r67519402>:
>
> > @@ -0,0 +1,9 @@
> > +## Two Truths and a Lie
> > +
> > +I wear a ring
> > +
> > +I can't log into eduroam wifi
> > +
> > +I'm going to California after this course
>
> Liar liar pants on fire
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/huangyz97/TwoTruths/pull/1/files/2336413ad492977681007d3ec369f335446a73c2#r67519402>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ATEc-pRqqg8bC-yEvNAzS3XsVOQbdMBpks5qMq_dgaJpZM4I4aPW>
> .
>